analyses/fastqc-analysis/README.md

```diff
@@ -0,0 +1,52 @@
+# Pipeline for FastQC quality control tool for high throughput sequence data analysis
+
+## Usage
+
+`run-fastqc-analysis` is designed to be run as if it was called from this module directory even when called from outside of this directory.
+
+Parameters according to the project and analysis strategy will need to be specified in the following scripts:
+- `run-fastqc-analysis.sh`: define path to input data with `*R2*.fastq.gz` files.
+
+### Run module on an interactive session on HPC
+
+To run the script in this module from the command line sequentially, use:
+
+```
+bash run-fastqc-analysis.sh
+```
+
+### Run module by using lsf on HPC
+
+There is also the option to run a lsf job on the HPC cluster by using the following command on an HPC node:
+
+```
+bsub < lsf_script.txt
```

Comment:
There is a typo. This command should be bsub < lsf-script.txt (intsead of lsf_script.txt)

Reply:
Thanks for noticing. Now fixed here 3fb7da09fea5f0267202ea1416dc06ba1005815e